app/models/document_capture_session.rb

```diff
@@ -17,6 +17,9 @@ def store_result_from_response(doc_auth_response)
     session_result.captured_at = Time.zone.now
     session_result.attention_with_barcode = doc_auth_response.attention_with_barcode?
     session_result.selfie_check_performed = doc_auth_response.selfie_check_performed?
+    session_result.doc_auth_success = doc_auth_response.doc_auth_success?
+    # nil(selfie not required) or true/false
+    session_result.selfie_success = doc_auth_response.selfie_success
```

Comment:
I wonder if we should be consistent here with the `?` for methods returning a boolean

Reply:
@night-jellyfish , it can be nil when selfie check is not performed.